client/src/worldgen.rs

```diff
@@ -384,21 +423,51 @@ impl Surface {
     }
 }
 
-fn trilerp<N: na::RealField>(
+fn triserp<N: na::RealField>(
     &[v000, v001, v010, v011, v100, v101, v110, v111]: &[N; 8],
     t: na::Vector3<N>,
+    threshold1: N,
+    threshold2: N,
```

Comment:
It looks like `threshold2` is *always* `1 - threshold1`, so it should probably be inferred rather than passed everywhere.

Reply:
Made the suggested changes.